sympy/parsing/autolev/__init__.py

```diff
@@ -0,0 +1,47 @@
+"""Used for parsing Autolev code to SymPy code."""
+
+from sympy.external import import_module
+
+def parse_autolev(inp, outFile=None):
```

Comment:
use a PEP8 linter and follow its suggestions, for example: `outFile` -> `out_file`.